.storybook/recipes/BaselineCard/BaselineCard.stories.tsx

```diff
@@ -8,14 +8,24 @@ export default {
   component: BaselineCard,
   args: {
     label: 'Selection of Evidence',
-    body: "In this Focus Area, you explore the main question: How does the structures of organisms enable life's functions?",
+    body: `In this Focus Area, you explore the main question: How does the structures of organisms enable life's functions?`,
```

Comment:
Just curious – why did you make this change?

Reply:
This was a relic from an earlier approach that I switched away from.